script.module.clouddrive.common/clouddrive/common/ui/addon.py

```diff
@@ -72,26 +70,33 @@ class CloudDriveAddon(RemoteProcessCallable):
     _export_progress_dialog_bg = None
     _system_monitor = None
     _video_file_extensions = ['mkv', 'mp4', 'avi', 'iso', 'nut', 'ogg', 'vivo', 'pva', 'nuv', 'nsv', 'nsa', 'fli', 'flc', 'wtv', 'flv']
```

Comment:
FYI If you are looking for Kodi supported formats; I like to use this:
```
MUSIC_EXTS         = (xbmc.getSupportedMedia('music')).split('|')  
IMAGE_EXTS         = (xbmc.getSupportedMedia('picture')).split('|')
MEDIA_EXTS         = (xbmc.getSupportedMedia('video')).split('|')
```

Reply:
That is great! I'll add it right away.